src/providers/ffz/FfzEmotes.cpp

```diff
@@ -79,8 +79,37 @@ namespace {
 
         return {Success, std::move(emotes)};
     }
-    std::pair<Outcome, EmoteMap> parseChannelEmotes(const QJsonObject &jsonRoot)
+    std::tuple<Outcome, EmoteMap, boost::optional<EmotePtr>>
+        parseChannelResponse(const QJsonObject &jsonRoot)
```

Comment:
Making a `parseChannelEmotes` and a `parseModBadge` function separately would work just as well. Here, The result is a tuple so need to either have comments or read the code to understand what the tuple items are.

Reply:
Also, unrelated to this PR, the Outcome in the tuple could be removed it is unused.